contributors.json

```diff
@@ -781,5 +781,12 @@
   "xxx32": {
     "country": "India",
     "name": "Aarushi"
+  },
+  "raj5036":{
```

Comment:
Order in ascending order by github user name

Reply:
Okay

Reply:
Is it okay now? @subeshb1